From 12024adbf37cb5c493add520fbfb7c26e8318007 Mon Sep 17 00:00:00 2001
From: Son Lam <son.lam.ym@renesas.com>
Date: Thu, 22 Aug 2019 21:06:41 +0700
Subject: [PATCH 286/405] arm64: dts: renesas: hihope-common: implement pinctrl
 group and pinctrl-names for SCIF2

This patch implements pinctrl group and pinctrl-names for SCIF2

Signed-off-by: Son Lam <son.lam.ym@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index 9a66b43..24811dd 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -268,6 +268,9 @@
 };
 
 &scif2 {
+	pinctrl-0 = <&scif2_pins>;
+	pinctrl-names = "default";
+
 	status = "okay";
 };
 
-- 
2.7.4

